docs/schema.md

```diff
@@ -262,6 +264,7 @@ It defines how this object will be handled. Each type has a different set of pro
 - `APPLICATION_UID`: The uuid of the created `Application` object.
 - `ISTIO_ENABLED`: Indicates whether Istio is enabled for the deployment.
 - `INGRESS_AVAILABLE`: Indicates whether the cluster is detected to have Ingress support.
+- `CERTIFICATE`: To be used to support a custom certificate or generate a self-signed certificate.
```

Comment:
TLS_CERTIFICATE

Reply:
Done, but I'm not sure about field under `type: TLS_CERTIFICATE`. Should it be (1) `certificate` or (2) `tlsCertificate`?

1)
```yaml
properties:
  certificate:
    type: string
    x-google-marketplace:
      type: TLS_CERTIFICATE
      certificate:
        generatedProperties:
          base64EncodedPrivateKey: TLS_CERTIFICATE_KEY
          base64EncodedCertificate: TLS_CERTIFICATE_CRT
```

2)
```yaml
properties:
  certificate:
    type: string
    x-google-marketplace:
      type: TLS_CERTIFICATE
      tlsCertificate:
        generatedProperties:
          base64EncodedPrivateKey: TLS_CERTIFICATE_KEY
          base64EncodedCertificate: TLS_CERTIFICATE_CRT
```

I think the option 2 is more clear, however it doesn't follow current convention. Please let me know, what do you think.

Reply:
I changed to `tlsCertificate`. PTAL